src/collection.jl

```diff
@@ -142,3 +152,43 @@ function images end
         @yield FITS(row.path)[row.hdu]
     end
 end
+
+
+@doc raw"""
+    process(f, df::DataFrame; path = nothing, save_prefix = nothing, save_suffix = nothing, save_delim = "_", ext = r"fits(\.tar\.gz)?"i)
+
+This is a function to apply multiple function `f` on all elements of data frame and then save it.
```

Comment:
"multiple"?  It's only one

Reply:
We can apply two functions, no?
```
function proc(f, x)
    println(f(x))
end

f1(x) = x + 2
f2(x) = x^2

proc(2) do x
    y = f1(x)
    z = f2(y)
end # outputs 16
```

Reply:
That's confusing: the entire body of the `do` block constitutes a single anonymous function, inside which one can do whatever they want.

Reply:
Hmm, true! It's only a single function. I will make the changes in documentation.